app/assets/javascripts/tasks_index.coffee

```diff
@@ -88,21 +89,43 @@ intializeAdvancedFilter =->
       $advancedFilterActive.val(false)
       $drop.removeClass('fa-caret-up').addClass('fa-caret-down')
     else
-      $search.css("border-bottom-left-radius","0px")
+      $search.css("border-bottom-left-radius", "0px")
       $advanced.css("border-bottom-right-radius", "0px")
       $advancedFilterActive.val(true)
       $drop.removeClass('fa-caret-down').addClass('fa-caret-up')
 
   if $('#order_param')
-    #    $('#' + order.value).addClass('active')
+#    $('#' + order.value).addClass('active')
```

Comment:
Can / should we remove this line? It doesn't seem to work, currently

Reply:
see below